worlds/animal_well/options.py

```diff
@@ -132,14 +132,49 @@ class WheelTricks(Choice):
     default = 0
 
 
-class WeirdTricks(Toggle):
+class BallThrowing(Choice):
     """
-    Include performing "weird" tricks in the logic.
-    Some of these tricks are difficult, tedious, or inconsistent.
-    Use at your own risk.
+    Include in logic using the ball to hit switches or buttons not ""designed"" for it.
+    Off means the ball will rarely be used for anything other than breaking blocks, spikes or guard shields.
+    Simple means the ball can be used to hit easy targets without any real rebound, setup or moving. Most ""yoyo chute"" buttons are included here.
+    Advanced means hitting your target may require bouncing off a wall, or moving while throwing to adjust momentum or get a good angle.
+    Expert includes any more complicated tricks, including those that require specific setups or getting lucky.
+    """									
+    internal_name = "ball_throwing"
+    display_name = "Ball Throwing"
+    option_off = 0
+    option_simple = 1
+    option_advanced = 2
+    option_expert = 3
+    default = 0
+
+
+class ObscureTricks(Toggle):
     """
-    internal_name = "weird_tricks"
-    display_name = "Weird Tricks"
+    Include a number of solutions to puzzles that are obscure or hard to understand.
+    These tricks aren't necessarily difficult to perform once you know what they are. If they are difficult, you must also turn on the respective difficulty or trick setting to enable it.
```

Comment:
```suggestion
    Include solutions to puzzles that are obscure or hard to understand in logic.
    These tricks aren't necessarily difficult to perform once you know what they are.
```
"aren't necessarily difficult" covers the case where they are difficult, so no need for that last sentence

Reply:
Not specifying how this interacts with difficulty-based tricks was a source of confusion when I laid out these options in the original spreadsheet. What do we think about

"These tricks won't be any harder to perform than any other tricks currently in logic once you know what they are."?

Reply:
That works

Reply:
Sounds great.